devstart

```diff
@@ -11,7 +11,10 @@ echo '        \/          \/     \/     \/                            \/
 echo ''
 
 echo "[MumukiDevstart::ClassroomApi] Installing dependencies...."
-bundle install --quiet
+bundle install
+
+echo "[MumukiDevstart::ClassroomApi] Starting MongoDB server...."
+mongod 2>&1 > /dev/null &
```

Comment:
I think that Mongo should be started outside this script because many of us do it as a service or with different configuration (like `dbpath`).